samples/sample_decode/src/sample_decode.cpp

```diff
@@ -272,6 +272,10 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         {
             pParams->bPerfMode = true;
         }
+        else if (0 == msdk_strcmp(strInput[i], MSDK_STRING("-powersaving")))
```

Comment:
Please add this key to sample help in this file, and to the documentation https://github.com/Intel-Media-SDK/MediaSDK/blob/master/doc/samples/readme-decode_linux.md